Match Android's Reader events

leandroalonso opened the pull request:
This PR adds some missing events introduced [in Android here](https://github.com/wordpress-mobile/WordPress-Android/pull/12818).

## To test

### Following

1. Tap Reader
2. Tap Following
3. Check that `reader_following_shown` is fired

### Saved

1. Tap Reader
2. Tap Saved
3. Check that `reader_saved_list_shown` is fired

### Liked

1. Tap Reader
2. Tap Liked
3. Check that `reader_liked_shown` is fired

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

leandroalonso committed:
Track reader following tab shown

leandroalonso committed:
Track Liked tab

leandroalonso committed:
Track liked tab tapped

yaelirub commented:
Hey @leandroalonso   👋 
A couple of questions:

1. This is probably not related to this PR but why am I seeing this:
<img width="769" alt="Screen Shot 2020-09-02 at 2 35 38 PM" src="https://user-images.githubusercontent.com/1335657/92039493-fcdceb00-ed29-11ea-86cc-d13ad334977c.png">

2. For Saved I see `reader_saved_list_viewed` not `reader_saved_list_shown` as described is that intentional?

leandroalonso committed:
Remove the old saved track

leandroalonso commented:
@yaelirub 

1. This is just a log that I added when a post is not deleted because it's associated with a card (this avoid a card being there without any post). But maybe it would create more problems and we don't really need it. I can remove, wdyt?

2. Fixed in af8e506137f881f273d7350ba76fd996a64db5d2

yaelirub commented:
>But maybe it would create more problems and we don't really need it

Yeah I think we can remove this one.

yaelirub reviewed (approved):
👍
